scalableminds/webknossos
# Loki for voxelytics logs

- Replaces elasticsearch with loki for vx log storage
- Changes the log frontend to only show the latest 1000 log entries

<img width="1635" alt="Screenshot 2023-01-20 at 20 09 03" src="https://user-images.githubusercontent.com/335438/213793316-bef894b0-d7ad-44ce-ac19-6c58e6465644.png">

### URL of deployed dev instance (used for testing):
- https://lokilogs.webknossos.xyz

### Steps to test:
- Run a vx workflow against https://lokilogs.webknossos.xyz and check the logs

------
(Please delete unneeded items, merge only when none are left open)
